screeps-game-api/src/objects/impls/room.rs

```diff
@@ -1,14 +1,34 @@
-use std::ops::Range;
+use std::{
+    marker::PhantomData,
+    mem,
+    ops::Range,
+};
 
-use stdweb::unstable::TryInto;
+use stdweb::{
+    Reference,
+    unstable::TryInto,
+};
 
-use {
-    constants::{Color, FindConstant, LookConstant, ReturnCode, StructureType, find::Exit},
-    HasPosition,
-    memory::MemoryReference,
-    objects::{Room, RoomPosition, StructureController, StructureStorage, StructureTerminal},
-    positions::LocalRoomName,
+use constants::{
+    Color, 
+    Direction,
+    find::Exit,
+    FindConstant, 
+    LookConstant, 
+    ReturnCode, 
+    StructureType, 
 };
+use HasPosition;
```

Comment:
If you feel like cleaning up the use style, this should be in the `objects::` group. It's probably not correct anywhere else anyways though until #49 is merged.